packages/app-builder/src/components/ui/Toolbar/ToolbarDropdown.tsx

```diff
@@ -0,0 +1,13 @@
+import { FormControl, InputLabel, Select } from '@material-ui/core'
+import React from 'react'
+
+export const ToolbarDropdown = ({ title, value, onChange, children }: any) => {
```

Comment:
interface TollBarDropdownProps {
  title: string
  value: any
  onChange: (value: any) => void
}

```suggestion
export const ToolbarDropdown: React.FC<TollBarDropdownProps> = ({ title, value, onChange, children }) => {
```